extensions/objects/templates/sanitized-url-editor.component.ts

```diff
@@ -46,15 +51,21 @@ export class SanitizedUrlEditorComponent {
   @Output() valueChanged = new EventEmitter();
   schema: SanitizedUrlSchema = {
     type: 'unicode',
-    validators: [{
-      id: 'is_nonempty'
-    }],
+    validators: [
+      {
+        id: 'is_nonempty'
+      },
+      {
+        id: 'is_regex_matched',
+        regexPattern: '(^https:\\/\\/.*)|(^(?!.*:\\/\\/)(.*))'
+      }
+    ],
     ui_config: {
       placeholder: 'https://www.example.com'
     }
   };
 
-  constructor(private changeDetectorRef: ChangeDetectorRef) {}
+  constructor(private changeDetectorRef: ChangeDetectorRef) { }
```

Comment:
Why is this changed?

Reply:
I think this space was added automatically by my code editor, would you want me to revert the change?

Reply:
Let's try to revert any changes that aren't needed!